src/OOP/Kata6/Person.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace OOP\Kata6;
+
+class Person
+{
+    protected string $name;
+    protected int $age;
+    protected string $occupation;
+
+    public function __construct($name, $age, $occupation)
+    {
+        $this->setName($name);
```

Comment:
It's better not to use internal class methods in constructor.
For a this Kata, better validate in costructor itself.